sljit_src/sljitNativeX86_common.c

```diff
@@ -681,6 +681,7 @@ static sljit_u8* detect_near_jump_type(struct sljit_jump *jump, sljit_u8 *code_p
 	sljit_s32 short_jump;
 	sljit_uw label_addr;
 	sljit_uw jump_addr;
+	sljit_uw rel_size;
```

Comment:
Probably you need to add `#if (defined SLJIT_CONFIG_X86_64 && SLJIT_CONFIG_X86_64)` arond this as well.